sdk/src/main/scala/com/horizen/account/state/ForgerStakeMsgProcessor.scala

```diff
@@ -68,6 +73,20 @@ case class ForgerStakeMsgProcessor(params: NetworkParams) extends FakeSmartContr
       throw new MessageProcessorInitializationException("initial tip already set")
 
     view.updateAccountStorage(contractAddress, LinkedListTipKey, LinkedListNullValue)
+
+    // forger list
+    /* Do not initialize it here since bootstrapping tool can not do the same as of now. That would result in
+       a different state root in the genesis block creation  */
+    val restrictForgerList = view.getAccountStorage(contractAddress, RestrictedForgerFlagsList)
+    if (!restrictForgerList.sameElements(NULL_HEX_STRING_32))
+      throw new MessageProcessorInitializationException("restrictForgerList already set")
+
+      /*
+    if (networkParams.restrictForgers) {
+      val forgersIndexesArray = new Array[Byte](networkParams.allowedForgersList.size)
+      view.updateAccountStorageBytes(contractAddress, RestrictedForgerFlagsList, forgersIndexesArray)
+    }
+     */
```

Comment:
I think, we will stay with the current approach, so the commented part can be removed.

Reply:
Ok, done